devlib/instrument/daq.py

```diff
@@ -156,10 +153,7 @@ def get_raw(self):
         return self._raw_files
 
     def teardown(self):
-        self.execute('close')
+        self.daq_client.close()
         if not self.keep_raw:
             if os.path.isdir(tempdir):
```

Comment:
It looks like this might have been an existing issue however `tempdir` is not defined here.

Reply:
Yes, this has been an issue since it was introduced in bb1552151a32e6d21f4c97763d73a071557661b5. I'll add a commit to fix it.

Reply:
Fixed in 01c5376